include/proxy/hdrs/MIME.h

```diff
@@ -1139,35 +1141,32 @@ class MIMEHdr : public HdrHeapSDKHandle
   int parse(MIMEParser *parser, const char **start, const char *end, bool must_copy_strs, bool eof, bool remove_ws_from_field_name,
             size_t max_hdr_field_size = UINT16_MAX);
 
-  int              value_get_index(const char *name, int name_length, const char *value, int value_length) const;
-  const char      *value_get(const char *name, int name_length, int *value_length) const;
-  std::string_view value_get(std::string_view const &name) const; // Convenience overload.
-  int32_t          value_get_int(const char *name, int name_length) const;
-  uint32_t         value_get_uint(const char *name, int name_length) const;
-  int64_t          value_get_int64(const char *name, int name_length) const;
-  time_t           value_get_date(const char *name, int name_length) const;
-  int              value_get_comma_list(const char *name, int name_length, StrList *list) const;
-
-  void value_set(const char *name, int name_length, const char *value, int value_length);
-  void value_set_int(const char *name, int name_length, int32_t value);
-  void value_set_uint(const char *name, int name_length, uint32_t value);
-  void value_set_int64(const char *name, int name_length, int64_t value);
-  void value_set_date(const char *name, int name_length, time_t value);
+  int              value_get_index(std::string_view name, std::string_view value) const;
+  std::string_view value_get(std::string_view name) const;
+  int32_t          value_get_int(std::string_view name) const;
+  uint32_t         value_get_uint(std::string_view name) const;
+  int64_t          value_get_int64(std::string_view name) const;
+  time_t           value_get_date(std::string_view name) const;
```

Comment:
I think for this one and a few others we could provide an override that takes string_view rather than replace the existing one.  That will avoid some of the conversions to string_view that just get converted back immediately.

Reply:
I thought we could change lower-level functions such as `mime_hdr_prepare_for_value_set` to take string_view once this pull request is merged.

Reply:
I wonder should I change lower-level functions to take string_view before this pull request?
Or should I revert removing the version taking `const char *` and `int` as you suggested?